config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -94,10 +94,49 @@ presubmits:
         resources:
           requests:
             memory: "6Gi"
-
+  # manual-release-bump-required
+  - name: pull-kops-e2e-kubernetes-aws-beta
```

Comment:
/shrug - because we have never forked -beta for kops-aws presubmit..

Reply:
ummmm... this is kubernetes/kops, which... probably shouldn't better

Reply:
done